### PR TITLE
Add client helper function for reading trx result

### PR DESCRIPTION
**Pull request by marcelka**

### CODE REVIEW

File: src/transactor.js [hunk 1]
```diff
@@ -150,11 +151,10 @@ export function transactor(firebase, handlers, todoTrxRef, closedTrxRef) {
             // here, so warn user about it)
             result = JSON.parse(JSON.stringify({result}))
           } catch (e) {
-            console.log(`Error for transaction id=${id}, result cannot be saved\n
-                        to firebase (result=${result})`)
+            console.log(`Error for transaction id=${id}, result cannot be saved\n to firebase (result=${result})`)
             result = {}
           }
-          set(closedTrxRef.child(trData.frbId), {data: trData, result})
+          set(closedTrxRef.child(trData.frbId), {data: trData, ...result})
```
Comment (tomaskulich) [inline src/transactor.js]: Why the splat? You may accidentally overwrite 'data' key.. is there any reason for this? However, I don't have a strong opinion on this one

File: src/client_useful.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import {DONE_TRX_PATH} from './settings'
+
+export function readTransactionResult(firebase, trxId, closedTrxPath) {
```
Comment (tomaskulich) [inline src/client_useful.js]: IMO closedTrxPath should have some sensible default value (and should be optional)

Ahhaa! it is! Why isn't this explicitly stated in the function signature?

File: src/transactor.js (hunk 1 above)
Comment (marcelka) [inline src/transactor.js]: No, I will not accidentally overwrite anything. As you can see above, result is either in the form {result: <handler-return-value>} or it is an empty map.

If I did not splat it here, I would get {result: {result: <handler-return-value}} or {result: {}}

File: src/client_useful.js (hunk 2 above)
Comment (marcelka) [inline src/client_useful.js]: :dart:

**Pull request closed without merging**